index.html

```diff
@@ -32,11 +32,6 @@ <h2>Pricing sections</h2>
 
```

Comment:
Always remember to close all open pace in your code. It will not stop your code from running but it does let your code looks cleaner.